libs/net/dal/Migrations/20230228191725_1.0.5.cs

```diff
@@ -57,7 +67,7 @@ protected override void Down(MigrationBuilder migrationBuilder)
                 nullable: false,
                 oldClrType: typeof(int),
                 oldType: "integer")
-                .OldAnnotation("Npgsql:ValueGenerationStrategy", NpgsqlValueGenerationStrategy.IdentityByDefaultColumn);
+                .OldAnnotation("Npgsql:ValueGenerationStrategy", NpgsqlValueGenerationStrategy.None);
```

Comment:
Had to manually change this to fix one issue with the rollback.  This may indicate a bug in the snapshot.